src/Kafka/Consumer.hs

```diff
@@ -48,7 +97,7 @@ import Kafka.Types                       as X
 
 -- | Runs high-level kafka consumer.
 -- A callback provided is expected to call 'pollMessage' when convenient.
-{-# DEPRECATED runConsumer "Use newConsumer/closeConsumer instead" #-}
+{-# DEPRECATED runConsumer "Use 'newConsumer'/'closeConsumer' instead" #-}
```

Comment:
An example where I added quotes to get correct links in Haddock, as such:
![image](https://user-images.githubusercontent.com/1204125/80006682-826fa500-84c5-11ea-9269-1c41225600ed.png)